### DISCUSSION TITLE
stringRedisTemplate.opsForValue().get(key) is null ？

### DISCUSSION THREAD

**Discussion by Breathtak** [redisson/redisson#6231] [Q&A] [answered]
I use RedissonAutoConfiguration's stringRedisTemplate bean
![1729154324474](https://github.com/user-attachments/assets/cbc4a45e-f959-45d2-a656-40b7b9e065db)
Then use stringRedisTemplate set value as follow:
![1729154533594](https://github.com/user-attachments/assets/34415449-9a7a-4e0d-8d2b-88d49d22e11c)
but redis has result:
![1729154616946](https://github.com/user-attachments/assets/3eb8b008-3edc-4c86-b294-4fe9bf20da46)

**Comment by mrniko** [comment, marked as answer]
try to set Redisson `readMode` setting to `MASTER`.
